From 46f81857f6d4ea2ec49d2cff8225e374c3ba17f0 Mon Sep 17 00:00:00 2001
From: freedesktop_sdk_updater <freedesktop_sdk_updater@libreml.com>
Date: Thu, 28 Nov 2024 17:02:36 +0000
Subject: [PATCH 2/2] Update elements/components/libjxl.bst to v0.11.1-0

Updates elements/components/libjxl.bst to v0.11.1-0 (794a5dc)
Previous version was v0.11.0-0 (4df1e9e)
---
 elements/components/libjxl.bst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/components/libjxl.bst b/elements/components/libjxl.bst
index 24f6654eff..3f6e46a760 100644
--- a/elements/components/libjxl.bst
+++ b/elements/components/libjxl.bst
@@ -52,11 +52,11 @@ variables:
     -DJPEGXL_ENABLE_DOXYGEN=ON
 
 sources:
 - kind: git_repo
   url: github:libjxl/libjxl.git
   track: v0.11.*
   exclude:
   - "*rc*"
   - "*snapshot*"
   - "*base*"
-  ref: v0.11.0-0-g4df1e9eccdf86b8df4c0c7c08f529263906f9c4f
+  ref: v0.11.1-0-g794a5dcf0d54f9f0b20d288a12e87afb91d20dfc
-- 
2.48.1

